config/i18n-tasks.yml

```diff
@@ -87,6 +87,16 @@ search:
   ## If `strict` is `false`, guess usages such as t("categories.#{category}.title"). The default is `true`.
   # strict: true
 
+  ## Allows adding ast_matchers for finding translations using the AST-scanners
+  ## The available matchers are:
+  ## - RailsModelMatcher
+  ##     Matches ActiveRecord translations like
+  ##     User.human_attribute_name(:email) and User.model_name.human
+  ##
+  ## To implement your own, please see `I18n::Tasks::Scanners::AstMatchers::BaseMatcher`.
+  <% require 'i18n/tasks/scanners/ast_matchers/rails_model_matcher' %>
+  <% I18n::Tasks.add_ast_matcher('I18n::Tasks::Scanners::AstMatchers::RailsModelMatcher') %>
```

Comment:
Which translations does this match?

Reply:
I wanted to check if it will match at least some of the unused keys, but it seems that it didn't and forgot to remove comment this back.

Reply:
Gotcha, let's remove it then.